scheduler/src/cook/components.clj

```diff
@@ -399,6 +416,8 @@
                    role)
      :mesos-framework-name (fnk [[:config [:mesos {framework-name "Cook"}]]]
                              framework-name)
+     :mesos-framework-id (fnk [[:config [:mesos {framework-id nil}]]]
```

Comment:
@sradack @shamsimam The reason to define two keys is that the cook startup code has 2 phases - in the first phase it translates the "literal" config to the config settings used downstream, and in the second phase, it uses these translated settings to actually bring up the components. `:mesos-framework-id` represents the value from phase 1, and `:framework-id` represents the value used in phase 2.